packages/fxa-customs-server/bin/customs_server.js

```diff
@@ -56,4 +56,4 @@ process.on('unhandledRejection', (err) => {
 
 const initPromise = init();
 
-export default initPromise;
+module.exports = initPromise;
```

Comment:
It would be better to fix the imports than changing this back to CommonJS.

Reply:
@StaberindeZA Do you know if we filed a follow up for getting this working ESM syntax? If not, I can file something.

Reply:
Oh, I thought this was already CommonJS and was just accidentally updated with ES modules syntax?

Reply:
@dschom nope I don't think so. Could you please file something?